src/pages/Home/components/CountrySummary/index.jsx

```diff
@@ -1,11 +1,18 @@
 import "./CountrySummary.scss"
+import moment from "moment";
+
 import BarChart from "../BarChart"
 
 function CountrySummary() {
   return (
     <div className="container country-summary-container">
       <h3 className="title">Comparação entre estados</h3>
-      <p className="data-description">Último Dia</p>
+      <p className="data-description">
+        Dados até:{' '}
+        <span className="featured-text">
+          {moment().format("D/M/YYYY")}
+        </span>
+      </p>
       <div className="chart-wrapper">
         <BarChart />
       </div>
```

Comment:
In this feature, it is important to show the date on which the last data update occurred.

Under normal conditions this is today's date, but it may happen that it is not.

You can obtain the value of the update date in the variable lastDate defined in the dependency "@bbjacob123/rt-chart" or in "@bbjacob123/rt-chart-inverted".

You will need to do something like this:

```JavaScript
import define from "@bbjacob123/rt-chart";

const lastUpdateRef= useRef(null)

runtime.module(define, name => {
    if (name === "lastDate") {
        return new Inspector(lastUpdateRef.current)
    }
}

<div ref={lastUpdateRef}></div>
```

Reply:
> In this feature, it is important to show the date on which the last data update occurred.
> 
> Under normal conditions this is today's date, but it may happen that it is not.
> 
> You can obtain the value of the update date in the variable lastDate defined in the dependency "@bbjacob123/rt-chart" or in "@bbjacob123/rt-chart-inverted".
> 
> You will need to do something like this:
> 
> ```js
> import define from "@bbjacob123/rt-chart";
> 
> const lastUpdateRef= useRef(null)
> 
> runtime.module(define, name => {
>     if (name === "lastDate") {
>         return new Inspector(lastUpdateRef.current)
>     }
> }
> 
> <div ref={lastUpdateRef}></div>
> ```

Great, thanks! o/

Reply:
Yes, it does

Reply:
I was the one to tell Marco to just use today's data. But you are right. This way is much more robust and better. I approve of Jeferson's suggestion. 👊